scipy/interpolate/_aaa.py

```diff
@@ -213,14 +220,17 @@ def __init__(self, points, values, *, rtol=None, max_terms=100):
         z, uni = np.unique(z, return_index=True)
         f = f[uni]
 
-        self._compute_weights(f, z, rtol, max_terms)
+        self._compute_weights(z, f, rtol, max_terms)
```

Comment:
What's going on here? Was it a bug in the original implementation (then can we have a test that would have caught it) or are you changing the signature here (for convenience?)

Reply:
No bug just for consistency with the order of `__init__`